python/graphstorm/eval/evaluator.py

```diff
@@ -1222,62 +820,17 @@ def compute_score(self, rankings, train=False): # pylint:disable=unused-argument
             return_metrics[metric] = return_metric.item()
         return return_metrics
 
-    def evaluate(self, val_scores, test_scores, total_iters):
-        """ GSgnnLinkPredictionModel.fit() will call this function to do user defined evalution.
-
-        Parameters
-        ----------
-        val_scores: dict of tensors
-            Rankings of positive scores of validation edges for each edge type.
-        test_scores: dict of tensors
-            Rankings of positive scores of test edges for each edge type..
-        total_iters: int
-            The current interation number.
-
-        Returns
-        -----------
-        val_mrr: float
-            Validation mrr
-        test_mrr: float
-            Test mrr
-        """
-        with th.no_grad():
-            if test_scores is not None:
-                test_score = self.compute_score(test_scores)
-            else:
-                test_score = {"mrr": "N/A"} # Dummy
-
-            if val_scores is not None:
-                val_score = self.compute_score(val_scores)
 
-                if get_rank() == 0:
-                    for metric in self.metric:
-                        # be careful whether > or < it might change per metric.
-                        if self.metrics_obj.metric_comparator[metric](
-                            self._best_val_score[metric], val_score[metric]):
-                            self._best_val_score[metric] = val_score[metric]
-                            self._best_test_score[metric] = test_score[metric]
-                            self._best_iter[metric] = total_iters
-            else:
-                val_score = {"mrr": "N/A"} # Dummy
-
-        return val_score, test_score
-
-
-class GSgnnPerEtypeMrrLPEvaluator(GSgnnMrrLPEvaluator):
+class GSgnnPerEtypeMrrLPEvaluator(GSgnnBaseEvaluator, GSgnnLPMrrEvalInterface):
```

Comment:
Why this one is GSgnnPerEtypeMrrLPEvaluator and the above one is GSgnnLPEvaluator?

Reply:
Changed `GSgnnLPEvaluator` to `GSgnnLPMrrEvalutor`.